src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -324,8 +327,17 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
       for (Entity entity : result.asIterable()) {
         String currTitle = (String) entity.getProperty("title");
         String currLocation = (String) entity.getProperty("location");
-        String currStartDate = entity.getProperty("startDate").toString();
-        String currEndDate = entity.getProperty("endDate").toString();
+        
+        if (entity.getProperty("startDate") == null)
```

Comment:
This is also debugging code? Are dates required?

Reply:
Yes, that was also debugging code, and I'm removing them right now. Dates are required, but I wanted to check if we had existing events in the database with invalid dates since they would cause problems with the sort feature.